README.md

```diff
@@ -191,7 +191,7 @@ If an error occurred, `nil` and a string describing the error will be returned.
 
 ### get\_result
 
-**syntax:** *uuid, matched_value, captures = c:get_result(matched_field)*
```

Comment:
We'd better explain what the `captures_len` means.

Reply:
added.